### PR TITLE
Added plugin nexus_ai

### PR DESCRIPTION
**Pull request by fiber-punk**

### CODE REVIEW

File: _plugins/nexus_ai.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+# Set this to true if your plugin uses the dependency_links setup parameter to include
+# library versions not yet published on pypi. SHOULD ONLY BE USED IF THERE IS NO OTHER OPTION!
+#follow_dependency_links: false
+
+tags:
+- secure
+- AI
+- Machine Learning
+- to your plugin
```
Comment (jneilliii) [inline _plugins/nexus_ai.md:30]: questionable tag, remove

File: _plugins/nexus_ai.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+id: nexus_ai
+title: Octoprint Nexus AI
+description: Nexus AI was oriniall developed for Sentry Pro to detect print failure. Fiberpunk  make it free and accessible to as many 3D printer users as possible. 
+author: Fiberpunk team
+#- first author name
+#- second autor name
+license: AGPLv3
+
+# today's date in format YYYY-MM-DD, e.g.
+date: 2020-09-29
```
Comment (jneilliii) [inline _plugins/nexus_ai.md:13]: correct release date

**Review by jneilliii — CHANGES_REQUESTED** [review]
couple of small changes required here, and then in your repo the GitHub user is incorrect [here](https://github.com/fiberpunk1/OctoPrint-Nexus-AI/blob/da7b398a1e3c84c2971af91c7291a583734d1426/octoprint_nexus_ai/__init__.py#L165), should be fiberpunk1 I think? 

Also, you need to create a release in your GitHub repo for the plugin that matches the version number specified in setup.py. Then for updating everyone's instance you update that version number in setup.py and create new release with same version number tag.

File: _plugins/nexus_ai.md (hunk 2 above)
Comment (fiber-punk) [reply _plugins/nexus_ai.md:13]: Thanks.
I will make modifications, and then send a  pull request with fiberpunk1.

**Comment by jneilliii** [comment]
closing, changes seem to be re-PR'd as #1111

**Pull request closed without merging**